common/views/components/SearchTabs/SearchTabs.tsx

```diff
@@ -290,6 +295,11 @@ const SearchTabs: FunctionComponent<Props> = ({
   ];
 
   function onTabClick(id: string) {
+    if(id === 'tab-images'){
```

Comment:
I am still a little confused why, when clicking a tab, we also want the form to submit?

Reply:
Say you've conducted a search in the catalogue api and have X results displayed, then switch tabs to images. The results update to show image results:

![tabs 2](https://user-images.githubusercontent.com/6051896/125958255-fcc1a3fd-3e94-43a3-b7fd-2171e7fd0f02.gif)

If we didn't submit the form again we would still be displaying the catalogue results, which I think would be confusing and is not how it currently works. 

@alicerichmond @DominiqueMarshall is the current behaviour, where the results update when the selected tab changes, expected?

Reply:
Yeah, that is expected behaviour as, like you mentioned, it would be really confusing if you switched tabs and the results didn't match the tab you selected.

Reply:
Super, thanks for explaining @gestchild - definitely wasn't querying the UX, just couldn't figure out why it was there... Tests / comments might help.